mono/metadata/object-internals.h

```diff
@@ -2331,4 +2333,7 @@ mono_gc_wbarrier_value_copy_internal (void* dest, /*const*/ void* src, int count
 void
 mono_gc_wbarrier_object_copy_internal (MonoObject* obj, MonoObject *src);
 
+#define MONO_BEGIN_UNSUSPENDABLE /* nothing yet */
```

Comment:
I think these will:
- be names `MONO_ENTER_NO_SAFEPOINTS`/`MONO_EXIT_NO_SAFEPOINTS`
- live in `mono/utils/mono-threads-api.h`